ui/src/app/components/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="dialog-container">
```

Comment:
Instead of adding a new component for confirm dialog we can rename and use the info dialog component, because the code and purpose is very similar.

Reply:
Confirm-dialog component is deleted and info-dialog is modified to accommodate both the feature.